src/platform/nrfconnect/DiagnosticDataProviderImplNrf.cpp

```diff
@@ -87,34 +87,52 @@ CHIP_ERROR DiagnosticDataProviderImplNrf::GetWiFiRssi(int8_t & rssi)
     return err;
 }
 
-// below will be implemented when the WiFi driver exposes Zephyr NET_STATISTICS API
 CHIP_ERROR DiagnosticDataProviderImplNrf::GetWiFiBeaconLostCount(uint32_t & beaconLostCount)
 {
-    return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
+    WiFiManager::NetworkStatistics stats;
```

Comment:
Can you also follow up: https://github.com/nrfconnect/sdk-connectedhomeip/commit/bf8b787ae6f7494ce8cebe4fa2b7eb13ed642b85 and make `GetWiFiInfo` return `SecurityTypeEnum` instead of making the cast above? And remove the TODO in line 60.

Reply:
Ok, fixed.

Reply:
Thanks!